public/css/style.css

```
@@ -1,210 +1,217 @@
-/* consent box */
 .elao-consent {
-    --default-elao-consent-primary: var(--elao-consent-primary, #007bff);
-    --default-elao-consent-primary-dark: var(--elao-consent-primary-dark, #256cdf);
-    --default-elao-consent-secondary: var(--elao-consent-secondary, #dfebfa);
-    --default-elao-consent-secondary-dark: var(--elao-consent-secondary-dark, #cadef8);
-    --default-elao-consent-danger: var(--elao-consent-danger, #ef4055);
-    --default-elao-consent-neutral: var(--elao-consent-neutral, #eae8e8);
-    --default-elao-consent-light: var(--elao-consent-light, #fff);
-    --default-elao-consent-dark: var(--elao-consent-dark, #252525);
-
-    padding-top: 30px;
-    width: 600px;
-    position: fixed;
-    bottom: 10px;
-    left: 10px;
-
-    max-height: calc(100vh - 20px);
-    overflow: auto;
-
-    color: var(--default-elao-consent-dark);
-    background-color: var(--default-elao-consent-light);
-    border: solid 1px var(--default-elao-consent-neutral);
-    border-radius: 15px;
-    z-index: 2147483647;
+  --default-elao-consent-primary: var(--elao-consent-primary, #007bff);
+  --default-elao-consent-primary-dark: var(--elao-consent-primary-dark, #256cdf);
+  --default-elao-consent-secondary: var(--elao-consent-secondary, #dfebfa);
+  --default-elao-consent-secondary-dark: var(--elao-consent-secondary-dark, #cadef8);
+  --default-elao-consent-danger: var(--elao-consent-danger, #ef4055);
+  --default-elao-consent-neutral: var(--elao-consent-neutral, #eae8e8);
+  --default-elao-consent-light: var(--elao-consent-light, #fff);
+  --default-elao-consent-dark: var(--elao-consent-dark, #252525);
+
+  padding-top: 20px;
+  width: 420px;
```

La largeur du bloc est un peu grande. On va tout réduire ici. 
```suggestion
  width: 350px;
  font-size: 14px;
```